library/utils.c

```diff
@@ -289,7 +289,16 @@ static void init_debug(uv_loop_t *loop) {
         tlsuv_set_debug(tlsuv_level, tlsuv_logger);
     }
 
-    ziti_debug_out = stderr;
+    char *logfile = getenv("ZITI_LOG_FILE");
+    if (logfile) {
```

Comment:
i don't love that this doesn't cover log rotation type policies. it also opens the file but never seems to close it? will that leak the descriptor when this process exits or will it get cleaned up?

Reply:
logrotation belongs outside of SDK, this is just an option for default logger. An application using SDK is free to override it

Reply:
all process resources are cleanup up on exit

Reply:
why provide it at all then? the sdk already has callback based functionality. seems like it should either be here and complete, or not and we let the consumer decide if it logs to  a file

Reply:
it is probably the next step (this is how it is done in other libraries -- openssl, etc) 
I want to keep this for now until downstream projects can implement it -- probably needs zitilib layer API